sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -67,6 +67,11 @@
       </dependency>
 
       <!-- Spring Cloud Azure -->
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>azure-spring-data-cosmos</artifactId>
+        <version>3.33.0-beta.1</version>
+      </dependency>
```

Comment:
Line 58 already has one for `azure-spring-data-cosmos`. Should we delete one?

Reply:
Done.